common/main/en.xml

```diff
@@ -265,6 +280,8 @@ annotations.
 			<language type="ig">Igbo</language>
 			<language type="ii">Sichuan Yi</language>
 			<language type="ik">Inupiaq</language>
+			<language type="ike">Eastern Canadian Inuktitut</language>
+			<language type="ikt">Western Canadian Inuktitut</language>
```

Comment:
When there are multiple language names in the ISO table how does CLDR decide which one? Should we add the alternates?

Reply:
We don't typically need alternates. It would only be if both names are used with the same frequency, and there is a large user community.